### ISSUE TITLE
Update to Bootstrap 5.x and SCSS

### ISSUE THREAD

**Issue by camtin** [springernature/pa11y-dashboard#307]
We should consider having the dashboard interface updated to use Bootstrap v5.3.2 or whatever the latest is.  The latest version of Bootstrap has considerable feature and accessibility improvements. SCSS has overtaken LESS in popularity and should be included as part of this updated.

Currently the site is using Bootstrap v3.0.1, which is 10 years old this month.
